File: Problems/SimpleMath_5/SimpleMath.py
while True:
  str1 = input("Enter first number: ")
  str2 = input("Enter second number: ")
  if str1.isnumeric() and str2.isnumeric(): 
    break
  else:
    print("Please enter two positive numbers")
  
num1 = int(str1)
num2 = int(str2)

add = num1 + num2
dif = num1 - num2
prod = num1 * num2
quot = num1 / num2

print("%i + %i = %i\n" % (num1, num2, add) + "%i - %i = %i\n" % (num1, num2, dif) + "%i * %i = %i\n" % (num1, num2, prod) + "%i / %i = %i\n" % (num1, num2, quot))

File: Problems/SimpleMath_5/SimpleMath.cpp
#include <string>
#include <iostream>
#include <stdlib.h>
#include <stdio.h>

using namespace std;

int main ()
{
    string str1, str2, endptr1, endptr2;
    double dbl1, dbl2;
    //while(true)
    //{
        cout << "Enter first number: ";
        cin >> str1;
        cout << "Enter second number: ";
        cin >> str2;
    //}
    dbl1 = atof(str1.c_str());
	dbl2 = atof(str2.c_str());

	double add = dbl1 + dbl2;
	double dif = dbl1 - dbl2;
	double prod = dbl1 * dbl2;
	double quot = dbl1 / dbl2;

	cout << dbl1 << " + " << dbl2 << " = " << add << "\n" << dbl1 << " - " << dbl2 << " = " << dif << "\n";
	cout << dbl1 << " * " << dbl2 << " = " << prod << "\n" << dbl1 << " / " << dbl2 << " = " << quot << "\n";
}

/*TO DO: Check the input to ensure that the input is a valid number*/
